main/src/cgeo/geocaching/maps/google/v2/GoogleMapActivity.java

```diff
@@ -252,7 +254,18 @@ protected void onActivityResult(final int requestCode, final int resultCode, fin
 
     @Override
     public void showFilterMenu(final View view) {
-        MapUtils.openFilterActivity(this, mapBase.getFilterContext(), mapBase.getCaches());
+        FilterUtils.openFilterActivity(this, mapBase.getFilterContext(), mapBase.getCaches());
     }
 
+    @Override
+    public boolean showFilterList(final View view) {
+        return FilterUtils.openFilterList(this, mapBase.getFilterContext());
+    }
+
+    @Override
+    public void refreshWithFilter(final GeocacheFilter filter) {
+        mapBase.getMapOptions().filterContext.set(filter);
+        MapUtils.filter(mapBase.getCaches(), mapBase.getMapOptions().filterContext);
+        mapBase.refreshMapData(true);
```

Comment:
Are you sure you want to pass "true" here? IIRC true means that the method assumes that circleDrawing has switched (something you can set in the map settings).

Reply:
Was not on purpose, I will change it